src/plotman/plotinfo.py

```diff
@@ -0,0 +1,108 @@
+import attr
+
+@attr.s(repr=True, init=False)
+class PlotInfo:
+    """Represents the results of a finished plot job"""
+    started_at: str = ""
+    plot_id: str = ""
+    buckets: int = 0
+    threads: int = 0
+    buffer: int = 0
+    plot_size: int = 0
+    tmp_dir1: str = ""
+    tmp_dir2: str = ""
+    phase1_duration_raw: float = 0
+    phase2_duration_raw: float = 0
+    phase3_duration_raw: float = 0
+    phase4_duration_raw: float = 0
+    total_time_raw: float = 0
+    copy_time_raw: float = 0
+    filename: str = ""
+
+    def is_empty(self) -> bool:
```

Comment:
Wouldn't this indicate it being incomplete, not empty?

```suggestion
    def is_incomplete(self) -> bool:
```